app/models/answer.rb

```diff
@@ -0,0 +1,5 @@
+class Answer < ApplicationRecord
+  belongs_to :question,
+  belongs_to :contact,
```

Comment:
should this maybe belong to profile instead?  we can always trace where the answer came from by looking at the questions virtual form

Reply:
That's a good point. I'll make the change.